src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.java

```diff
@@ -882,6 +901,27 @@ private KfsVendorDataWrapper createInsuranceCertificateNote(PaymentWorksVendor p
         }
         return kfsVendorDataWrapper;
     }
+    
+    protected String findPoCountryToUse(PaymentWorksVendor pmwVendor) {
+        String poCountry = pmwVendor.getPoCountry();
+        String poCountryUsCanadaAustraliaOther = pmwVendor.getPoCountryUsCanadaAustraliaOther();
+        String poCountryToUse;
+        if (paymentWorksFormModeService.shouldUseForeignFormProcessingMode()) {
+            if (StringUtils.equalsIgnoreCase(poCountryUsCanadaAustraliaOther, PaymentWorksConstants.PO_ADDRESS_COUNTRY_OTHER) ||
+                    StringUtils.isBlank(poCountryUsCanadaAustraliaOther)) {
+                poCountryToUse = poCountry;
+            } else {
+                poCountryToUse = poCountryUsCanadaAustraliaOther;
+            }
+        } else {
+            if (StringUtils.isNotBlank(poCountry)) {
```

Comment:
Is this fixing a bug with domestic?

Reply:
Yes, if poCountry has a value, use that, otherwise use poCountryUsCanadaAustraliaOther.  With the mapping fixed, I expect poCountry to be empty and have the value in  poCountryUsCanadaAustraliaOther.  However, there might be some iterations of data and form details were poCountry has a value, and that value should take precedence

Reply:
Thank you for explaining this.